xhr/xhr-authorization-redirect.any.js

```diff
@@ -0,0 +1,28 @@
+// META: global=window,worker
```

Comment:
I think worker here should be replaced with sharedworked,dedicatedworker as we cant have XHR requests in the context of service workers? 
This causes the following error:
```
PROMISE_REJECT(object "ReferenceError: XMLHttpRequest is not defined")
```

Reply:
Yeah that makes sense. Thanks for debugging!